assets/terraform/gce/os.tf

```diff
@@ -10,17 +10,20 @@ variable "oss" {
 
   default = {
     # os -> {project}/{image}
-    "ubuntu:16"     = "ubuntu-os-cloud/ubuntu-1604-xenial-v20190816"
-    "ubuntu:18"     = "ubuntu-os-cloud/ubuntu-1804-bionic-v20190813a"
-    "ubuntu:latest" = "ubuntu-os-cloud/ubuntu-1804-bionic-v20190813a"
-    "redhat:7"      = "rhel-cloud/rhel-7-v20190813"
-    "centos:7"      = "centos-cloud/centos-7-v20190813"
-    "debian:8"      = "debian-cloud/debian-8-jessie-v20180401"
-    "debian:9"      = "debian-cloud/debian-9-stretch-v20190813"
-    "debian:latest" = "debian-cloud/debian-9-stretch-v20190813"
-    "suse:12"       = "suse-cloud/sles-12-sp4-v20190617"
-    "suse:15"       = "suse-cloud/sles-15-sp1-v20190828"
-    "suse:latest"   = "suse-cloud/sles-15-sp1-v20190828"
+    "ubuntu:16"     = "ubuntu-os-cloud/ubuntu-1604-xenial-v20200407"
+    "ubuntu:18"     = "ubuntu-os-cloud/ubuntu-1804-bionic-v20200414"
+    "ubuntu:19"     = "ubuntu-os-cloud/ubuntu-1904-eoan-v20200413a"
+    "ubuntu:latest" = "ubuntu-os-cloud/ubuntu-1904-eoan-v20200413a"
```

Comment:
I don't believe Gravity officially supports 19.04: https://gravitational.com/gravity/docs/requirements/.  Do we have a reason to test on it (e.g. a customer using it)?

Since focal released today, I'd swap this out to 20.04, which we will support as the next Ubuntu LTS.